src/Symfony/Bridge/Twig/ErrorRenderer/TwigErrorRenderer.php

```diff
@@ -41,17 +41,17 @@ public function __construct(Environment $twig, HtmlErrorRenderer $fallbackErrorR
 
     public function render(\Throwable $exception): FlattenException
     {
-        $exception = $this->fallbackErrorRenderer->render($exception);
-        $debug = \is_bool($this->debug) ? $this->debug : ($this->debug)($exception);
+        $flattenException = $this->fallbackErrorRenderer->createFlattenException($exception);
```

Comment:
a new conflict rule needs to be added in the twig-bridge package to forbid installing it with symfony/error-handler <6.2, which would not have this method.

Reply:
Added.